### ISSUE TITLE
Pegasus workflow management for extremely large jobs (near or above 6 digit task count)

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#484]
### What would you like to see added?

Also "genetic simulations at scale". More generally, how to scale up arbitrary workflows to many, many CPU hours or tasks.

Examples: https://pegasus.isi.edu/application-showcase/
Blake specific example: https://pegasus.isi.edu/application-showcase/inference-of-human-demographic-history-with-approximate-bayesian-computation/ and https://pegasus.isi.edu/2018/12/10/pegasus-helped-ecology-and-evolutionary-biology-graduate-student-at-the-university-of-arizona-shed-light-on-human-population-history/
Paper: https://www.softxjournal.com/article/S2352-7110(18)30121-3/fulltext

Supplementary data showing when/why this is needed: https://www.softxjournal.com/article/S2352-7110(18)30121-3/fulltext#supplementaryMaterial

Bottom line: Executed 5.1 million CPU hours in 5 months for a single researcher.
